templates/home/index.html.twig

```diff
@@ -1,7 +1,59 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Student{% endblock %}
+
+    {% block body %}
+        <ul class="list-group m-4">
+            <li class="list-group-item list-group-item-info">Studentai</li>
+            {% for student in someVariable %}
+                <li class="list-group-item">
+                    <a href="{{ path('student') }}?name={{ student.student|url_encode }}&project={{ student.project|url_encode }}">{{ student.student|escape }}</a>
```

Comment:
```suggestion
                    <a href="{{ path('student', {'name': student.student, 'project': student.project}) }}">{{ student.student}}</a>
```
`path` funkcija gali paimti argumentus

Plačiau:
 * https://symfony.com/doc/current/routing.html#generating-urls-in-javascript